docs/working-with-projects/project-configuration/configure-webhooks.md

```diff
@@ -9,81 +9,120 @@ The action that is taken upon receiving a notification from a Harbor project dep
+when you select the Slack type, and fill a Slack incoming webhook URL as endpoint, the message you received in Slack will be like,
+```json
+Harbor webhook events
+event_type: PUSH_ARTIFACT
+occur_at: April 15th at 11:59 AM
+operator: admin
+event_data:
+{
+    "resources": [
+        {
+            "digest": "sha256:8a9e9863dbb6e10edb5adfe917c00da84e1700fa76e7ed02476aa6e6fb8ee0d8",
+            "tag": "latest",
+            "resource_url": "hub.harbor.com/test-webhook/debian:latest"
+        }
+    ],
+    "repository": {
+        "date_created": 1586922308,
+        "name": "debian",
+        "namespace": "test-webhook",
+        "repo_full_name": "test-webhook/debian",
+        "repo_type": "private"
     }
-  ]
 }
 ```
 
 ### Webhook Endpoint Recommendations
 
-The endpoint that receives the webhook should ideally have a webhook listener that is capable of interpreting the payload and acting upon the information it contains. For example, running a shell script.
+There are two kinds of endpoints.  For `HTTP` the endpoint that receives the webhook should ideally have a webhook listener that is capable of interpreting the payload and acting upon the information it contains. For example, running a shell script.
+
+And for Slack endpoint, you should follow the [guide of Slack incoming webhook](https://api.slack.com/messaging/webhooks).
 
 ### Example Use Cases
 
 You can configure your continuous integration and development infrastructure so that it performs the following types of operations when it receives a webhook notification from Harbor.
 
-- Image push: 
+- Artifact push: 
   - Trigger a new build immediately following a push on selected repositories or tags.
-  - Notify services or applications that use the image that a new image is available and pull it.
-  - Scan the image using Clair.
-  - Replicate the image to remote registries.
+  - Notify services or applications that use the artifact that a new artifact is available and pull it.
+  - Scan the artifact using Clair.
+  - Replicate the artifact to remote registries.
 - Image scanning:
   - If a vulnerability is found, rescan the image or replicate it to another registry.
   - If the scan passes, deploy the image.
 
 ### Configure Webhooks
 
 1. Log in to the Harbor interface with an account that has at least project administrator privileges.
+
 1. Go to **Projects**, select a project, and select **Webhooks**.
 
-    ![Webhooks option](../../../img/webhooks1.png)
+    ![Webhooks option](../../img/webhooks1.png)
+
+1. Select notify type `HTTP`, so the webhook will be send to a HTTP endpoint.
+
+1. Select events that you want to subscribe.
 
 1. Enter the URL for your webhook endpoint listener.
+
 1. If your webhook listener implements authentication, enter the authentication header. 
+
 1. To implement `HTTPS POST` instead of `HTTP POST`, select the **Verifiy Remote Certficate** check box.
 
-    ![Webhook URL](../../../img/webhooks2.png)
+    ![Webhook URL](../../img/webhooks2.png)
```

Comment:
'../../../' is correct, this is a behavior of the system that hooks up to the website, basically the img is in its own directory, this needs to be changed back or image won't show properly on website